src/common/snippets/include/snippets/op/subgraph.hpp

```diff
@@ -143,13 +142,15 @@ class Subgraph : public ov::op::util::SubGraphOp {
     // Return estimated unique buffer count (upper bound). It's needed for tokenization
     static auto get_estimated_buffer_count(const ov::NodeVector& ops) -> size_t;
     static auto is_domain_sensitive_op(const std::shared_ptr<ov::Node>& op) -> bool;
+    std::shared_ptr<lowered::LinearIR>
+    convert_body_to_linear_ir(const std::shared_ptr<IShapeInferSnippetsFactory>& factory = std::make_shared<IShapeInferSnippetsFactory>());
```

Comment:
I deliberately removed `const` because we will have store relevant shapeInfer instance.
I will bring it back here if you insist, but remove again in [18563](https://github.com/openvinotoolkit/openvino/pull/18563) 😁